### ISSUE TITLE
rasa interactive doesnt work

### ISSUE THREAD

**Issue by nicolasfarina** [RasaHQ/rasa#4658]
![a](https://user-images.githubusercontent.com/52745496/67306435-54d18100-f4cd-11e9-9db2-ef244d38a198.png)

**Rasa version**:1.4.1

**Python version**: 2.7.4

**Operating system** (windows, osx, ...): Ubuntu

**Comment by sara-tagger** [comment]
Thanks for the issue, @federicotdn will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗
###### Caw! 🐧

**Comment by dsmackie** [comment]
This issue appears to stem from the requirement that the argument `persist_nlu_data` exists when calling the training function, but it is not being set to None when calling training via interactive mode.

https://github.com/RasaHQ/rasa/blob/33daa89648222fec73c276eda8f1d4b61e985619/rasa/cli/interactive.py#L54-L64

Can be corrected by adding `args.persist_nlu_data = None` to the top of the interactive method.

I would think that `interactive_core` would require the same amendment.

**Comment by dsmackie** [comment]
Adding to my last, making the above change does allow interactive mode to proceed but results in further errors as shown below. I can not confirm if they are related or due to a previous commit.

```
Bot loaded. Visualisation at http://localhost:5006/visualization.html.
Type a message and press enter (press 'Ctr-c' to exit).
Processed Story Blocks: 100%|████████████████| 18/18 [00:00<00:00, 3095.30it/s, # trackers=1]
2019-10-23 21:16:09 ERROR    rasa.core.training.interactive  - An exception occurred while recording messages.
Traceback (most recent call last):
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/interactive.py", line 1396, in record_messages
    await _plot_trackers(sender_ids, plot_file, endpoint)
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/interactive.py", line 1330, in _plot_trackers
    event_sequences[-1], event_sequences, output_file=None, max_history=2
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/visualization.py", line 486, in visualize_neighborhood
    _merge_equivalent_nodes(graph, max_history)
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/visualization.py", line 207, in _merge_equivalent_nodes
    graph, i, j, max_history
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/visualization.py", line 147, in _nodes_are_equivalent
    return graph.node[node_a]["label"] == graph.node[node_b]["label"] and (
AttributeError: 'MultiDiGraph' object has no attribute 'node'
2019-10-23 21:16:09 ERROR    asyncio  - Task exception was never retrieved
future: <Task finished coro=<_serve_application.<locals>.run_interactive_io() done, defined at /home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/interactive.py:1449> exception=AttributeError("'MultiDiGraph' object has no attribute 'node'",)>
Traceback (most recent call last):
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/interactive.py", line 1456, in run_interactive_io
    sender_id=uuid.uuid4().hex,
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/interactive.py", line 1396, in record_messages
    await _plot_trackers(sender_ids, plot_file, endpoint)
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/interactive.py", line 1330, in _plot_trackers
    event_sequences[-1], event_sequences, output_file=None, max_history=2
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/visualization.py", line 486, in visualize_neighborhood
    _merge_equivalent_nodes(graph, max_history)
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/visualization.py", line 207, in _merge_equivalent_nodes
    graph, i, j, max_history
  File "/home/rasa/chattabox-rasa/env/lib/python3.6/site-packages/rasa/core/training/visualization.py", line 147, in _nodes_are_equivalent
    return graph.node[node_a]["label"] == graph.node[node_b]["label"] and (
AttributeError: 'MultiDiGraph' object has no attribute 'node'
```

**Comment by federicotdn** [comment]
Hi @nicolasfarina, thanks for bringing this up. This issue has been fixed in https://github.com/RasaHQ/rasa/commit/525a2443ec6863081ec7774a1a707272f687c944, but the fixed hasn't been released yet. It will be included in 1.4.2. I'll leave this issue open until that's released.
